integration/helpers/node-template/package.json

```diff
@@ -5,9 +5,9 @@
   "sideEffects": false,
   "type": "module",
   "scripts": {
-    "build": "node ./node_modules/@react-router/dev/dist/cli.js build",
-    "dev": "node ./node_modules/@react-router/dev/dist/cli.js dev",
-    "start": "node ./node_modules/@react-router/serve/dist/cli.js ./build/index.js",
+    "build": "react-router build",
+    "dev": "react-router dev",
+    "start": "react-router-serve ./build/server/index.js",
```

Comment:
This error was a bit buried with the old verbose script usage. Found it while ensuring these scripts work when attempting to manually debug tests.